src/renderer/dx/DxRenderer.cpp

```diff
@@ -41,6 +41,7 @@ DxEngine::DxEngine() :
     _displaySizePixels{ 0 },
     _foregroundColor{ 0 },
     _backgroundColor{ 0 },
+    _selectionBackground{ 0 },

```

Comment:
Should `_selectionBackground` be `DEFAULT_FOREGROUND` by default?

Note that `OpenConsole.exe` (conhost) can run with the Dx engine as well (by setting `UseDx=1` in the registry under `HKCU\Console`), where it won't have someone setting the selection background.

Reply:
If the Selection Background is being set to the Foreground colour by default, there should probably be a Selection Foreground colour, which would be set to the Background colour by default too.

Reply:
I'm mostly concerned that the default value of this variable has changed from `DEFAULT_FOREGROUND` to `0`. We should probably just file a follow up issue to track that work.

Reply:
Forgot about conhost being able to run with DxEngine! I agree, `_selectionBackground` should be `DEFAULT_FOREGROUND` here.

I'll file an issue to track selection foreground.